docs/sources/panels-visualizations/visualizations/flame-graph/index.md

```diff
@@ -19,61 +19,115 @@ weight: 100
 
 # Flame graph
 
-Flame graphs let you visualize profiling data. Using this visualization, the profile can be represented as a flame graph, table, or both
+Flame graphs let you visualize [profiling](https://grafana.com/docs/pyroscope/latest/introduction/profiling/) data. Using this visualization, a profile can be represented as a [flame graph](#flame-graph-mode), [top table](#top-table-mode), or both.
```

Comment:
You might also consider these two additional links: 
* Flame graphs: Visualizing performance data, https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/flamegraphs/
* Understand profile types and their uses in Pyroscope, https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/ (explains the  different profilers)

Reply:
Thank you, @knylander-grafana. I'll add these links!